twilio_integration/twilio_integration/api.py

```diff
@@ -114,3 +116,20 @@ def get_contact_details(phone):
 		'email_id': contact_doc.email_id,
 		'phone_number': contact_doc.phone
 	}
+
+@frappe.whitelist(allow_guest=True)
+def incoming_whatsapp_message(**kwargs):
+	args = frappe._dict(kwargs)
+	incoming_message(args)
+	resp = MessagingResponse()
+
+	# Add a message
+	resp.message(frappe.db.get_single_value('Twilio Settings', 'reply_message'))
+	return Response(resp.to_xml(), mimetype='text/xml')
+
+@frappe.whitelist(allow_guest=True)
+def whatsapp_message_status(**kwargs):
```

Comment:
Please add docstrings. Is this the endpoint where Twilio pushes message status details? better name please.